src/content/docs/distributed-tracing/infinite-tracing/set-trace-observer.mdx

```diff
@@ -176,7 +176,7 @@ If you already created a trace observer while setting up another type of service
 If you haven't already set up a trace observer, complete the following:
 
 1. Go to [one.newrelic.com](https://one.newrelic.com) and click **Apps**, and then under **Your apps**, click **New Relic Edge**.
-2. Select an account in the upper-left dropdown. If you have access to multiple accounts, make sure you're in the account where you want Infinite Tracing enabled. Since Tracing is a cross-account feature, the trace observers are limited to one observer per region per [account family](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/parent-child-account-structure/).
+2. Select an account in the upper-left dropdown. If you have access to multiple accounts, make sure you're in the account where you want Infinite Tracing enabled. If you can't add an observer, it's likely due to there being only [one observer allowed per region per account family](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#infinite-architecture).
```

Comment:
Consider simplifying like this: "...it's probably because only one observer is allowed per region, per account family."